doc/theoretical_description_classification.rst

```diff
@@ -6,15 +6,179 @@
+  corresponding :math:`k^{th}` fold removed.
+
+- The corresponding *out-of-fold* conformity score is computed for each :math:`i^{th}` point 
+
+- Compare the conformity scores of training instances with the scores of each label for each new test point in order to
+  decide whether or not the label should be included in the prediction set. 
+  For the APS method, the prediction set is constructed as follows (see equation 11 of [3]) : 
+
+.. math:: 
+    C_{n, \alpha}(X_{n+1}) = 
+    \Big\{ y \in \mathcal{Y} : \sum_{i=1}^n {\rm 1} \Big[ E(X_i, Y_i, U_i; \hat{\pi}^{k(i)}) < E(X_{n+1}, y, U_{n+1}; \hat{\pi}^{k(i)}) \Big] < (1-\alpha)(n+1) \Big\}
+
+where : 
+
+- :math:`E(X_i, Y_i, U_i; \hat{\pi}^{k(i)})` is the conformity score of training instance :math:`i`
+
+- :math:`E(X_{n+1}, y, U_{n+1}; \hat{\pi}^{k(i)})` is the conformity score of label :math:`y` from a new test point.
+
+
+
+
+.. The :class:`mapie.regression.MapieClassifier` class implements several conformal methods
+.. for estimating predictions sets, i.e. a set of possibilities that include the true label
+.. with a given confidence level.
+.. The full-conformal methods being computationally intractable, we will focus on the split-
+.. and cross-conformal methods. 
+
+.. Before describing the methods, let's briefly present the mathematical setting.
+.. For a classification problem in a standard independent and identically distributed
+.. (i.i.d) case, our training data :math:`(X, Y) = \{(x_1, y_1), \ldots, (x_n, y_n)\}`
+.. has an unknown distribution :math:`P_{X, Y}`. 
+
+.. Given some target quantile :math:`\alpha` or associated target coverage level :math:`1-\alpha`,
+.. we aim at constructing a set of possible labels :math:`\hat{T}_{n, \alpha} \in {1, ..., K}`
+.. for a new feature vector :math:`X_{n+1}` such that 
+
+.. .. math:: 
+..     P \{Y_{n+1} \in \hat{T}_{n, \alpha}(X_{n+1}) \} \geq 1 - \alpha
+
+
+.. 1. Split-conformal method
+.. -------------------------
+
+.. - In order to estimate prediction sets, one needs to "calibrate" so-called conformity scores
+..   on a given calibration set. The alpha-quantile of these conformity scores is then estimated
+..   and compared with the conformity scores of new test points output by the base model to assess
+..   whether a label must be included in the prediction set
+
+.. - The split-conformal methodology can be summarized in the scheme below : 
+..     - The training set is first split into a training set and a calibration set
+..     - The training set is used for training the model
+..     - The calibration set is only used for getting distribution of conformity scores output by
+..       the model trained only on the training set. 
+
+
+.. 2. The "score" method
+.. ---------------------
+
+.. 3. The "cumulated score" method
+.. -------------------------------
+
+.. 4. The cross-conformal method
+.. -----------------------------
+
+
+
+.. TO BE CONTINUED
+
+References
```

Comment:
Note that references seems to be a sub-part of 4. Split- and cross-conformal methods.

Reply:
Changed.